compiler/luci/lang/include/luci/IR/Nodes/CircleConv2D.h

```diff
@@ -34,7 +34,8 @@ namespace luci
  */
 class CircleConv2D final : public FixedArityNode<3, CircleNodeImpl<CircleOpcode::CONV_2D>>,
                            public CircleNodeMixin<CircleNodeTrait::FusedActFunc>,
-                           public CircleNodeMixin<CircleNodeTrait::Bias>
+                           public CircleNodeMixin<CircleNodeTrait::Bias>,
+                           public CircleNodeMixin<CircleNodeTrait::WeightCompression>
```

Comment:
Q) why does Conv2D have this attribute? why not the filter Constant?

Reply:
> Q) why does Conv2D have this attribute? why not the filter Constant?

CircleConst is virtual node - and we need somehow export and import circle - so I suggest to set this attribute when importing op from circle and also set this in CircleConst

Reply:
I don't understand. Please give more explanation.